es_loader/es_loader.py

```diff
@@ -41,12 +72,49 @@ def _create_index(self):
         self.es.indices.create(self.index_name)
         logger.debug('Index created')
 
+    def _get_job(self):
+        """Get job from ScrapingHubClient istance."""
+        self.job = self.sc.get_job(self.job_id)
```

Comment:
This method is very small and seems to be only used once, so I would inline it (just use this code and remove the method). If you want extra clarity, renamve `sc` to `scrapinghub_client` so it will be very explicit - `self.scrapinhub_client.get_job(self.job_id)`.